### PR TITLE
Add joining disclaimer content [feature flag]

### PR DESCRIPTION
**Pull request by ravikumar1002**
### What is the change?

 - I have added the ```dev=true``` feature flag for joining the disclaimer  before developer joining  form
 - Added utils.js script file In join.js file that is missing from

### Is Development Tested?
  Yes

### Before :
image

### After :
image

### Video of feature flag: 

https://github.com/Real-Dev-Squad/website-www/assets/86847625/477c7ebd-5fb7-478d-b18a-f22c109d354c

### CODE REVIEW

File: js/join.js [hunk 1]
```diff
@@ -39,6 +42,19 @@ const flowState = {
...
+const buttonContainer = document.querySelector('.button-container');
+const disclaimerCheckbox = document.getElementById('disclaimer-checkbox');
+const disclaimerFeatureFlag = document.querySelector('.disclaimer_feat-flag');
+
+window.addEventListener('load', (event) => {
+  if (queryString.includes('dev=true')) {
+    buttonContainer.classList.add('button-container_disclaimer');
+    disclaimerFeatureFlag.classList.remove('hidden');
+    startBtn.classList.remove('button-filled');
+    startBtn.classList.add('button-disabled');
```
Comment (kotesh-arya) [inline js/join.js:54]: You're adding the class 'button-disabled' here. May I know why there is a need to again add a `startBtn.disabled = true`

**Review by kotesh-arya — COMMENTED** [review]
Added a comment, please check 👍🏼

File: js/join.js (hunk 1 above)
Comment (ravikumar1002) [reply js/join.js:54]: Both have different terms :

Class `button-disabled` is used to give CSS styling when a button is disabled

`startBtn.disabled = true` is an attribute that is used to disable the button, if it is true then the user can not click on this button

**Review by kotesh-arya — APPROVED** [review]

**Comment by shubhamsigdar1** [comment]
Can you please add one video
In which you are showing feature behind feature flag

**Comment by ravikumar1002** [comment]
> Can you please add one video In which you are showing feature behind the feature flag and how your feature working
> 
> You done great work

Thanks Shubham, and I have also added a video of feature flag working video.

**Review by shubhamsigdar1 — APPROVED** [review]

**Review by rohan09-raj — APPROVED** [review]

**Pull request merged**